### DISCUSSION TITLE
LlamaFolio - changes requested

### DISCUSSION THREAD

**Discussion by 0xsign** [WalletConnect/walletconnect-monorepo#3633] [Developer Support]
Hey, I created a project and the submission got rejected for this reason:

`
"Dapp Testing - Could not verify if testing has been done on this dapp. As we do not have available V2 metric data for Sign. Please test your integration with our test wallet and submit for approval if successful. Link: https://react-web3wallet.vercel.app"
`

I did test it with this app and successfully connected, could you help me move forward ? I'm not sure what's required here.

Thanks

**Comment by glitch-txs** [comment]
Hi did you test signing the message?
